sdk/storage/azure-storage-blob/src/main/java/com/azure/storage/blob/models/StorageAccountInfo.java

```diff
@@ -35,27 +35,33 @@ public StorageAccountInfo(final SkuName skuName, final AccountKind accountKind)
     public StorageAccountInfo(final SkuName skuName, final AccountKind accountKind, Boolean isHnsEnabled) {
         this.skuName = skuName;
         this.accountKind = accountKind;
-        this.isHnsEnabled = isHnsEnabled;
+        this.isHnsEnabled = isHnsEnabled != null && isHnsEnabled;
```

Comment:
you can do `isHnsEnabled == Boolean.TRUE`

Reply:
Tried doing this but had to revert changes since it caused linting errors in our pipelines. Thanks for the suggestion though :)